subprojects/gradle/buildchecks/src/test/kotlin/com/avito/android/plugin/build_param_check/incremental_check/IncrementalKaptTaskTest.kt

```diff
@@ -0,0 +1,97 @@
+package com.avito.android.plugin.build_param_check.incremental_check
+
+import com.avito.test.gradle.AndroidAppModule
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.gradlew
+import org.junit.jupiter.api.Test
+
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+
+internal class IncrementalKaptTaskTest {
+
+    @Test
+    fun `build success with no warnings - unsupported Java version`(@TempDir projectDir: File) {
+        generateProject("none", projectDir)
```

Comment:
What is this string for? Does it change any logic?

Reply:
Do you mean `"none"` string? If yes then it changes the mode of incremental KAPT build check. Different tests use different modes.